jaxrs/src/main/java/org/killbill/billing/jaxrs/resources/AccountResource.java

```diff
@@ -333,7 +333,7 @@ public Response createAccount(final AccountJson json,
 
         final AccountData data = json.toAccountData();
         final Account account = accountUserApi.createAccount(data, context.createContext(createdBy, reason, comment, request));
-        return uriBuilder.buildResponse(uriInfo, AccountResource.class, "getAccount", account.getId());
+        return uriBuilder.buildResponse(uriInfo, AccountResource.class, "getAccount", account.getId(), jaxrsConfig.getJaxrsReturnPathLikeUrl());
```

Comment:
Instead of modifying the signature for `JaxrsUriBuilder #buildResponse` and then having to change the callers, would not it be simpler to inject the `JaxrsConfig` into  `JaxrsUriBuilder`. It looks like `JaxrsUriBuilder` is already injected through Guice (e.g in [AccountResource ](https://github.com/killbill/killbill/blob/master/jaxrs/src/main/java/org/killbill/billing/jaxrs/resources/AccountResource.java#L151)), however we miss the eager singleton binding in [DefaultJaxrsModule](https://github.com/killbill/killbill/blob/master/jaxrs/src/main/java/org/killbill/billing/jaxrs/glue/DefaultJaxrsModule.java#L35).